app/web/features/messages/requests/HostRequestSendField.tsx

```diff
@@ -72,6 +72,153 @@ const StyledHelpTextContainer = styled("div")(({ theme }) => ({
   marginBottom: theme.spacing(2),
 }));
 
+function ResponseGuideLinks({
+  isPast,
+  isHost,
+  status,
+}: {
+  isPast: boolean;
+  isHost: boolean;
+  status: HostRequestStatus;
+}) {
+  const isHostPending =
+    !isPast &&
+    isHost &&
+    status === HostRequestStatus.HOST_REQUEST_STATUS_PENDING;
+
```

Comment:
Would you mind moving these to separate components and importing them in here? That's generally the "React way". Also then you can just see an overview of what's happening in the parent component and click through to the children if needed. And then maybe you want to adjust the names `HostRequestRespondButtons` and `HostRequestGuideLinks` or whatever.

Reply:
Will do

Reply:
Done!